spec/latest/core/index.html

```diff
@@ -1193,6 +1193,36 @@ <h3>Templated Links</h3>
       </dl>
     </aside>
 
+    <p>The example <i>Description of an IRI Template</i> that was already mentioned
+      uses a <i>variableRepresentation</i> on the whole <i>IriTemplate</i>, but
+      each mapped variable represented as an <i>IriTemplateMapping</i> can have
+      it's own <i>variableRepresentation</i> declaration which SHOULD override
+      the declaration from the <i>IriTemplate</i> level for that very
+      <i>IriTemplateMapping</i> as it takes precedence. In case no no upper
+      declaration is available, the <i>IriTemplateMapping</i> one SHOULD be in force,
+      as it is shown on the example below.
```

Comment:
I think the last sentence is redundant or even the entire paragraph sounds slightly complicated.

There is always a variable representation of the entire template. Implicitly, it's `BasicRepresentation`. So, for any given variable:

1. Use variable's `variableRepresentation`
2. If not given, use the template's `variableRepresentation`
3. If not given, use `BasicReprresentation`

I don't think we must make the wording much more verbose than that

Reply:
Ok, you're right. I've replaced last sentence completely.